### DISCUSSION TITLE
v2.2.0

### DISCUSSION THREAD

**Discussion by keshav-space** [keshav-space/safenotes#114] [Announcements]
## What's Changed
* Support biometric authentication by @keshav-space in https://github.com/keshav-space/safenotes/pull/99
* Auto-detect text direction RTL/LTR by @keshav-space in https://github.com/keshav-space/safenotes/pull/103
* Support compact note overview by @keshav-space in https://github.com/keshav-space/safenotes/pull/105
* New dark/light theme picker by @keshav-space in https://github.com/keshav-space/safenotes/pull/108
* New translations by @comradekingu, @TheDarkKRONOS, @ersen0, @dperruso and @Atalanttore  in https://github.com/keshav-space/safenotes/pull/90, https://github.com/keshav-space/safenotes/pull/92, https://github.com/keshav-space/safenotes/pull/94,, https://github.com/keshav-space/safenotes/pull/96, https://github.com/keshav-space/safenotes/pull/98, https://github.com/keshav-space/safenotes/pull/100, https://github.com/keshav-space/safenotes/pull/101, https://github.com/keshav-space/safenotes/pull/106, https://github.com/keshav-space/safenotes/pull/107, https://github.com/keshav-space/safenotes/pull/109, https://github.com/keshav-space/safenotes/pull/111 and https://github.com/keshav-space/safenotes/pull/113

## New Contributors
* @Atalanttore made their first contribution in https://github.com/keshav-space/safenotes/pull/111
* @TheDarkKRONOS made their first contribution in https://github.com/keshav-space/safenotes/pull/107
* @dperruso made their first contribution in https://github.com/keshav-space/safenotes/pull/98
* @ersen0 made their first contribution in https://github.com/keshav-space/safenotes/pull/94

**Full Changelog**: https://github.com/keshav-space/safenotes/compare/v2.1.0...v2.2.0

`Note: If not sure of your device architecture use safenotes-v2.2.0-all.apk`

<hr /><em>This discussion was created from the release <a href='https://github.com/keshav-space/safenotes/releases/tag/'>v2.2.0</a>.</em>

**Comment by hallothere803** [comment]
can i be an english to indonesian translator for your application?

**Reply by keshav-space** [reply]
> can i be an english to indonesian translator for your application?

That would be awesome @nature-modren  :)

**Reply by hallothere803** [reply]
Done bro😅👍

**Reply by keshav-space** [reply]
> Done bro😅👍

Well, that was fast! 😳

Thank you! 🙇‍♂️
I'm merging [this PR](https://github.com/keshav-space/safenotes/pull/128) the new language will be included in next upcoming release.

**Reply by keshav-space** [reply]
@nature-modren I was preparing for v2.3 release, seems like we have some missing translations for Indonesian language.
``` python
    "No route: {route}": "",
    "Location: {locationPath}": "",
    "Last Backup: {lastBackupTime}": "",
    "{argsType}, Needed for route: {route}": "",
    "File saved at '{exportFileFullPathName}'": "",
```

**Reply by hallothere803** [reply]
I have completed the missing words ,maybe I missed haha😅👍

**Reply by keshav-space** [reply]
Thanks again :)

**Reply by hallothere803** [reply]
> Thanks again 
that's good....... but when the latest version is released I'm really waiting for it, a small update like changing the theme color, can change the arrangement of text, can change the font, or the like is an idea That's good, I'll wait for the latest update :)

**Reply by hallothere803** [reply]
When 2.3.0?

**Reply by keshav-space** [reply]
> When 2.3.0?

This month hopefully 🤞
